contrib/libtpms/template.py

```diff
@@ -0,0 +1,32 @@
+pkgname = "libtpms"
+pkgver = "0.9.6"
+pkgrel = 0
+build_style = "gnu_configure"
+make_cmd = "gmake"
+hostmakedepends = [
+    "automake",
+    "gmake",
+    "libtool",
+    "pkgconf",
+]
+makedepends = [
+    "openssl-devel",
+]
+checkdepends = [
+    "bash",
```

Comment:
when there's just one thing you can leave it as a single line instead of multiple